tests/efi-vars-editor-vm

```diff
@@ -33,8 +33,6 @@ lxc storage create "${poolName}" "${poolDriver}"
 
 echo "==> Create VM and boot"
 lxc launch "${IMAGE}" v1 --vm -s "${poolName}"
-waitInstanceReady v1
```

Comment:
This needs to stay (although with a comment would help) as otherwise lxd tries to access the nvram whilst qemu guest initializing it (git history may show more details)

Reply:
Indeed, found commit a9d485147ca1cdc04 and added a comment, thanks!